packages/containers/src/Filter/Filter.selectors.js

```diff
@@ -0,0 +1,25 @@
+import { DEFAULT_STATE, DISPLAY_NAME } from './Filter.container';
+
+/**
+ * Selector on the state from the filter id given.
+ * @param {object} state
+ * @param {string} idComponent
+ */
+export function getComponentState(state, idComponent) {
+	if (
+		state.cmf.components.has(DISPLAY_NAME) &&
+		state.cmf.components.get(DISPLAY_NAME).has(idComponent)
```

Comment:
This can be simplified by using [hasIn](https://facebook.github.io/immutable-js/docs/#/Map/hasIn).